api/src/routes/stardust/nftDetails/get.ts

```diff
@@ -27,6 +28,8 @@ export async function get(
     if (networkConfig.protocolVersion !== STARDUST) {
         return {};
     }
-
-    return StardustTangleHelper.nftDetails(networkConfig, request.nftId);
+    if (process.env.fakeNftDetails === "true") {
```

Comment:
Make the environment variable follow the usual naming convention, so `FAKE_NFT_DETAILS`

Reply:
Done